searchtool/server/boot/search.js

```diff
@@ -70,7 +71,8 @@ exports.buildSearch = function (req, res) {
                     pagein:paginate({totalItem:body.response.numFound, itemPerPage:20, currentPage:currentPage, url:'/newsearch',params:{q:q}}),
                     took:humanize.numberFormat(body.responseHeader.QTime,0 ),
                     highlighting:body.highlighting,
-                    term:q
+                    term:q,
+                    ptab: ptab
```

Comment:
Can we change this parameter to `dataset`, too? The way it is now is simple for just the two datasets, but will not work once we add more.